kubectl-plugin/pkg/cmd/create/create_cluster.go

```diff
@@ -119,22 +119,21 @@ func (options *CreateClusterOptions) Run(ctx context.Context, factory cmdutil.Fa
 
 	// Will generate yaml file
 	rayClusterObject := generation.RayClusterYamlObject{
-		Namespace:      *options.configFlags.Namespace,
-		ClusterName:    options.clusterName,
-		RayVersion:     options.rayVersion,
-		Image:          options.image,
-		HeadCPU:        options.headCPU,
-		HeadMemory:     options.headMemory,
-		WorkerGrpName:  options.workerGrpName,
-		WorkerReplicas: options.workerReplicas,
-		WorkerCPU:      options.workerCPU,
-		WorkerMemory:   options.workerMemory,
+		Namespace:   *options.configFlags.Namespace,
+		ClusterName: options.clusterName,
+		RayClusterSpecObject: generation.RayClusterSpecObject{
+			RayVersion:     options.rayVersion,
+			Image:          options.image,
+			HeadCPU:        options.headCPU,
+			HeadMemory:     options.headMemory,
+			WorkerGrpName:  options.workerGrpName,
+			WorkerReplicas: options.workerReplicas,
+			WorkerCPU:      options.workerCPU,
```

Comment:
Can you add WorkerGPU in a follow-up PR?

Reply:
I will make another ray kubectl plugin umbrella bug that will have the followups.

Reply:
I think specifying worker GPUs should probaby be in the beta criteria, it's pretty common with Ray